microsoft_atp/plugin.spec.yaml

```diff
@@ -581,89 +549,37 @@ actions:
         type: string
         title: Action ID
         description: Action ID
+        example: ffd1f0cb-68ad-44ea-bf90-d01061b965ec
         required: true
     output:
       machine_action_response:
-        type: machine_action_response
+        type: machine_action
         title: Machine Action Response
         description: A response that includes the result of the action, and supplemental
           information about the action taken
         required: true
 triggers:
-  get_alert_for_id:
```

Comment:
Does Get Alerts Matching Key supersede all these?

Reply:
Sort of...